<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceMangerWindow</class>
 <widget class="QDialog" name="DeviceMangerWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QToolButton" name="btnDevice">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReTurnMenu">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>10</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>780</width>
     <height>501</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="btnDevFind">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="btnDelete">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>170</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
